Sources/Penny/Penny.swift

```diff
@@ -9,12 +10,34 @@ struct Penny {
     }
 
     static func start(mainService: any MainService) async throws {
-        try await mainService.bootstrapLoggingSystem()
+        /// Use `1` instead of `System.coreCount`.
+        /// This is preferred for apps that primarily use structured concurrency.
+        let httpClient = HTTPClient(
+            eventLoopGroup: MultiThreadedEventLoopGroup.singleton,
+            configuration: .forPenny
+        )
+        let awsClient = AWSClient(httpClient: httpClient)
 
-        let bot = try await mainService.makeBot()
+        /// These shutdown calls are only useful for tests where we call `Penny.main()` repeatedly
+        defer {
+            /// Shutdown in reverse order of dependance.
+            try! awsClient.syncShutdown()
```

Comment:
We should not be doing this - this is a Swift bug in that it's not showing you a warning but these methods are (or should be) marked with `noasync` - we need to call the async shutdowns, which unfortunately means no defers

Reply:
Removed all `syncShutdown()`, let's see how the tests go.